dev/build.image/build.gradle

```diff
@@ -76,21 +76,35 @@ task copyLicenseToBuildImage (type:Copy) {
            tokens: [BUILD_ID: bnd.buildID, LIBERTY_VERSION: bnd.libertyRelease])
 }
 
+
 task copySwidTagToBuildImage (type:Copy) {
     dependsOn jar
     from project.file('publish')
     into project.file('wlp/lib/versions/tags')
-    include 'ibm.com_Open_Liberty*.swidtag'
+    include 'ibm.com_Open_Liberty-22.*.swidtag'
```

Comment:
why was `-22` added to this include?

Reply:
After discussion, out of the 3 choices: 
Change 22 -> 2, to delay any problems with version going from 22->23 next year 
Change the ibm.com_Open_Liberty-release.swidtag to something that doesnt interfere with the product tag task, since this tag template shouldnt change between versions
Look into a better way of pattern matching (not sure if we can use anything apart from ?, *, and ** with gradle)

Option number 2 was chosen and implemented in "Renamed fixpack tag so product task does not copy" commit